src/Libero/ruleset.xml

```diff
@@ -62,6 +62,9 @@
     </rule>
     <rule ref="Squiz.Classes.ClassFileName"/>
     <rule ref="Squiz.PHP.InnerFunctions"/>
+    <rule ref="Squiz.Strings.DoubleQuoteUsage">
+        <exclude name="Squiz.Strings.DoubleQuoteUsage.ContainsVar"/>
```

Comment:
Could do with feedback. I find them simpler to read/write, but it's limited what you can do and behaviour is odd sometimes (https://stackoverflow.com/a/43437427).